docs/cli/graphql-transformer/auth.md

```diff
@@ -158,7 +158,7 @@ Next, let's say that you wanted to modify the schema to allow only the owner of
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [create, delete] }]) {
+  @auth(rules: [{ allow: owner, operations: [delete] }]) {
```

Comment:
This is in conflict with line no. 141 & 148. We should verify the requirement of the "create" operation.

Reply:
updated to put create back, I think anyone can create it's not owner specific so it's confusing. it can be in the list or not and its permissions are the same is my understanding.